lib/util/serialization.js

```diff
@@ -73,6 +73,11 @@ module.exports = {
 		return getBinaryMiddleware().MEASURE_END_OPERATION;
 	},
 	get buffersSerializer() {
+		throw new Error(
+			"buffersSerializer was removed in favor of createBuffersSerializer"
+		);
+	},
+	createBuffersSerializer(hashFunction) {
```

Comment:
```suggestion
	createBuffersSerializer({ hashFunction = "xxhash64 }) {
```

for exensibility and defaults